app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+def go_to_cafe(friends: list, cafe: str) -> str:
+    mask_to_buy = 0
+    for friend in friends:
+        name_cafe = Cafe(cafe)
+        try:
+            name_cafe.visit_cafe(friend)
```

Comment:
So you don't need name_cafe variable just call the method from instance : `cafe.visit_cafe(friend)`